packages/core/src/command/hl7-notification/hl7-notification-webhook-sender-direct.ts

```diff
@@ -79,6 +90,24 @@ export class Hl7NotificationWebhookSenderDirect implements Hl7NotificationWebhoo
     });
     log(`Conversion complete and patient entry added`);
 
+    const clinicalInformation = this.extractClinicalInformation(newEncounterData);
+
+    log(`Writing TCM encounter to DB...`);
+    await this.writeTcmEncounterToDb(
```

Comment:
let's please remove "db" from the name, abstracting away where it's stored - maybe call it `storeTcmEncounter`?